linux/src/threadpool_linux.c

```diff
@@ -501,29 +531,156 @@ int threadpool_schedule_work(THREADPOOL_HANDLE threadpool, THREADPOOL_WORK_FUNCT
 
 int threadpool_timer_start(THREADPOOL_HANDLE threadpool, uint32_t start_delay_ms, uint32_t timer_period_ms, THREADPOOL_WORK_FUNCTION work_function, void* work_function_ctx, TIMER_INSTANCE_HANDLE* timer_handle)
```

Comment:
>threadpool_timer_start [](http://example.com/codeflow?start=4&length=22)

why isn't this function returning a NULL when it fails or otherwise a non-NULL value when it doesn't?